<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>593</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>High Speed</family>
    <pointsize>28</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <widget class="QWidget" name="MenuPage1">
    <widget class="QLabel" name="author_title">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>550</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Speed</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>By: @leodenglovescode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settings_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>340</y>
       <width>511</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createserver_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>511</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Server</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>440</y>
       <width>511</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="menu_title">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>571</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Speed</family>
       <pointsize>26</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>CS2 Server Manager</string>
     </property>
    </widget>
    <widget class="QLabel" name="version_title">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Speed</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Version Beta Test 1.0.0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manageservers_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>511</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Speed</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Manage Servers</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="CreateServerStep1">
    <widget class="QLabel" name="installsteamcmd_title">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>521</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Install Steamcmd</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="installsteamcmd_button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>491</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Speed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Install From Steam</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testinternet_button">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Speed</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Test Internet Connection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="continue_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>430</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Speed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HelpPage"/>
   <widget class="QWidget" name="SettingsPage"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
